osu! Mapping Olympiad #2 Results (#2475)
ppy/osu-wiki: news/2019-09-07-osu!taiko-mapping-olympiad-2-results.md

---
layout: post
title: "osu! Mapping Olympiad #2 Results"
date: 2019-09-07 17:00:00 +0000
---

> The judging for the first osu!taiko Mapping Olympiad contest has been completed and the results are in: congratulations to the first **osu!taiko Mapping Olympian** and taiko's newest *Elite Mapper*!

Reviewer: ```suggestion
The judging for the first [osu!taiko Mapping Olympiad contest](https://osu.ppy.sh/community/contests/65) has been completed, and the results are in: congratulations to **[Nofool](https://osu.ppy.sh/users/672430)**, the first osu!taiko Mapping Olympian and our newest *Elite Mapper*!
```
clickbaity title if u dont mention the winner

> [![](https://osu.ppy.sh/wiki/shared/news/banners/osu_mapping_olympiad.jpg)](https://osu.ppy.sh/community/contests/65)

Reviewer: ```suggestion
![](https://osu.ppy.sh/wiki/shared/news/banners/osu_mapping_olympiad.jpg)
```
banners don't have links anymore

> ![](https://osu.ppy.sh/wiki/shared/news/banners/osu_mapping_olympiad.jpg)

Reviewer 2: Link can be shortened.
```suggestion
![](/wiki/shared/news/banners/osu_mapping_olympiad.jpg)
```

> For the inaugural taiko Olympiad contest, we tasked mappers to create a full spread showcasing their proficiency in taiko mapping on the song *Bwa !* from [**Thaehan**](https://osu.ppy.sh/beatmaps/artists/7). There were a total of 33 entries and below are the results for the **top 10 entries**:

Reviewer: ```suggestion
For the inaugural [osu!taiko Mapping Olympiad contest](https://osu.ppy.sh/community/contests/65), we tasked mappers to create a full spread showcasing their proficiency in osu!taiko mapping with the song *Bwa !* from [Thaehan](https://osu.ppy.sh/beatmaps/artists/7). There were a total of 33 entries, and below are the results for the **top 10**:
```

> | Rank 	| Username  	| Entry Name          	| Flow and Structure (30) 	| Relevance (30) 	| Difficulty Spread (30) 	| Creativity (30) 	| Judge Impression (30) 	| Score 	|

Reviewer: shouldn't use whitespace to align tables (it doesn't really help anyway when the rows are so long). only use one space between content and the pipes

> |------	|-----------	|---------------------	|-------------------------	|----------------	|------------------------	|-----------------	|-----------------------	|-------	|

Reviewer: ```suggestion
| :-: | :-- | :-- | :-- | :-- | :-- | :-- | :-- | :-- |
```

> | #1   	| Nofool    	| Verifiable Monkey   	| 23.5                    	| 23             	| 26                     	| 22              	| 25                    	| 119.5 	|

Reviewer: probably link the usernames in here ( & include flags?)

Author: Only added the links to usernames but didn't added flags cause imo it would be too much for the table

| #4   	| Nishizumi 	| Monumental Chipmunk 	| 20.5                    	| 22.2           	| 18.2                   	| 18.8            	| 18                    	| 97.7  	|
| #5   	| [R]       	| Wrong Bat           	| 21.2                    	| 23             	| 22.5                   	| 8               	| 19                    	| 93.7  	|
| #6   	| sing216   	| Average Elephant    	| 20                      	| 22.9           	| 18                     	| 14.2            	| 18                    	| 93.1  	|
| #7   	| -Kazu-    	| Intelligent Lion    	| 21.5                    	| 18.5           	| 21.5                   	| 15              	| 15.5                  	| 92    	|
| #8   	| Zetera    	| Robust Whale        	| 19                      	| 21             	| 18.5                   	| 18              	| 15                    	| 91.5  	|
| #9   	| Fallmorph 	| Normal Fox          	| 19.5                    	| 20.3           	| 19.5                   	| 15              	| 17                    	| 91.3  	|
| #10  	| Sinnoh    	| Idealistic Opossum  	| 20.8                    	| 22             	| 15.5                   	| 14.5            	| 17.5                  	| 90.3  	|

> In **first place** is [Nofool](https://osu.ppy.sh/users/672430) with a score of **119.5**. Their entry received the best score from judge incandescence, as well as the overall 'Judging Impression' score! Overall an excellent entry across the board. Congratulations!

Reviewer: ```suggestion
In **first place** is [Nofool](https://osu.ppy.sh/users/672430) with a score of **119.5**. Their entry received the best score from judge incandescence, as well as the highest overall "Judge Impression" score! This an excellent entry across the board. Congratulations!
```

> Following close behind in **second place** is [Nifty](https://osu.ppy.sh/users/4956097) who received a total of **117.5** points. They earned the best scoring in the 'Flow and Structure' and 'Creativity' categories, not to mention they received an impressive overall score of 44/50 by judge Nepuri!

Reviewer: ```suggestion
Following close behind in **second place** is [Nifty](https://osu.ppy.sh/users/4956097), who received a total of **117.5** points. They earned the best scoring in the "Flow and Structure" and "Creativity" categories, not to mention they received an impressive overall score of 44/50 from judge Nepuri!
```

> [Volta](https://osu.ppy.sh/users/4154071) closes the podium by earning **third place** with a score of **106.3**. Their entry showcased a mastery of difficulty spread, receiving a score of 27/30 which was the highest among all entries.

Reviewer: ```suggestion
[Volta](https://osu.ppy.sh/users/4154071) closes the podium by earning **third place** with a score of **106.3**. Their entry showcases a mastery of difficulty spread, receiving a score of 27/30 in the category, beating out all other entries.
```

 
You can see all of the scores and comments from the judges via [this spreadsheet](https://docs.google.com/spreadsheets/d/1BwQLmpSGvprFBMh29lSfqzxPfD2QZLMGTPYNkoDiYTE/edit?usp=sharing), and you can [click here to check out all of the entries](https://drive.google.com/open?id=1c_fgvvq1skoqqDs47ZVuAqn8bIIW79mY)!
 
Congratulations to all of the top three finishers! Here is what each person will receive as their reward (as a reminder, 6 Olympiad Points are needed for one badge level):
 

> - **First place**: 3 months of osu!supporter, 6 Olympiad Points (Results in 'Mapping Olympian' badge), *Elite Mapper* title

Reviewer: ```suggestion
- **First place**: 3 months of osu!supporter, 6 Olympiad Points (results in a "Mapping Olympian" badge), *Elite Mapper* title
```
